tests/test_create_output.bats

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bats
+
+load 'test_fixtures'
+
+@test "create_output=True" {
+    export INPUT_CREATE_OUTPUT=True
+    run ${ENTRYPOINT_SH}
+    assert_output --partial "Will create JSON output"
+    assert_output --partial "Using verbosity level: -1"
```

Comment:
How much of a pain will it be here if we change all the verbosity levels in the validator itself?

Reply:
This should already be present, I believe. Although `-1` is not possible. Verbosity is currently set via repetetive additions of `-v`.

Reply:
Or I might not be following you completely here... Could you elaborate on what you mean?

Reply:
I just meant if I'm writing a PR that shifts all verbosities so that 0 is the new -1, will you be annoyed that you have you change all your tests that match for -1 :P

Reply:
Let's cross that bridge when we get to it. Luckily if it's only the tests that are affected, this shouldn't be a real issue.